daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -1998,6 +1998,124 @@ def _resume_device(self):
+               # Breath
+               'set_logo_breath_random_naga_hex_v2',
+               'set_logo_breath_single_naga_hex_v2',
+               'set_logo_breath_dual_naga_hex_v2',
+               # Static
+               'set_logo_static_naga_hex_v2',
+               # None
+               'set_logo_none_naga_hex_v2',
+               # Scroll wheel
+               'get_scroll_brightness', 'set_scroll_brightness',
+               # Spectrum
+               'set_scroll_spectrum_naga_hex_v2',
+               # Reactive
+               'set_scroll_reactive_naga_hex_v2',
+               # Breath
+               'set_scroll_breath_random_naga_hex_v2',
+               'set_scroll_breath_random_naga_hex_v2',
+               'set_scroll_breath_dual_naga_hex_v2',
+               # Static
+               'set_scroll_static_naga_hex_v2',
+               # None
+               'set_scroll_none_naga_hex_v2',
+               'get_left_brightness', 'set_left_brightness', 'get_right_brightness', 'set_right_brightness',
+               # Left side
+               'set_left_wave', 'set_left_static', 'set_left_spectrum', 'set_left_none', 'set_left_reactive', 'set_left_breath_random', 'set_left_breath_single', 'set_left_breath_dual',
+               # Right side
+               'set_right_wave', 'set_right_static', 'set_right_spectrum', 'set_right_none', 'set_right_reactive', 'set_right_breath_random', 'set_right_breath_single', 'set_right_breath_dual',
+
+               # Can set LOGO and Scroll with custom
+               'set_custom_effect', 'set_key_row']
+
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1590/1590_basilisk_ultimate.png"
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
+        "top_img": DEVICE_IMAGE
+    }
+
+    DPI_MAX = 20000
+
+    def _suspend_device(self):
+        """
+        Suspend the device
+        Get the current brightness level, store it for later and then set the brightness to 0
+        """
+        self.suspend_args.clear()
+        self.suspend_args['brightness'] = (_da_get_logo_brightness(self), _da_get_scroll_brightness(
+            self), _get_left_brightness(self), _get_right_brightness(self))
+
+        # Todo make it context?
+        self.disable_notify = True
+        _da_set_logo_brightness(self, 0)
+        _da_set_scroll_brightness(self, 0)
+        _set_left_brightness(self, 0)
+        _set_right_brightness(self, 0)
+        self.disable_notify = False
+
+    def _resume_device(self):
+        """
+        Resume the device
+        Get the last known brightness and then set the brightness
+        """
+        logo_brightness = self.suspend_args.get('brightness', (100, 100))[0]
+        scroll_brightness = self.suspend_args.get('brightness', (100, 100))[1]
+        left_row_brightness = self.suspend_args.get(
+            'brightness', (100, 100))[2]
+        right_row_brightness = self.suspend_args.get(
+            'brightness', (100, 100))[3]
```

Comment:
This block was also [updated](78f4c185821ad0636f0c5ac53c6b2615927b447a) since the initial PR correcting suspend/resume issues affecting similar mice. Should be updated as the following:

https://github.com/openrazer/openrazer/blob/b33acd80bfa5ec13d4885356e45f4ec022d6651d/daemon/openrazer_daemon/hardware/mouse.py#L112-L115

Reply:
fixed